Apps/phproofpoint/proofpoint_connector.py

```diff
@@ -269,17 +284,16 @@ def _on_poll(self, param):
         start_at = ((datetime.utcnow() - timedelta(minutes=mins))
                     .replace(microsecond=0).isoformat() + 'Z')
 
-        if (not self._state or 'last_poll' not in self._state or
-                self._state['last_poll'] < start_at):
+        if (not self._state or 'last_poll' not in self._state or self._state['last_poll'] < start_at):
             self._state['last_poll'] = start_at
 
         params = {
             'sinceTime': self._state['last_poll']
         }
 
         # Connect to the server
-        ret_val, data = self._make_rest_call(action_result,
-                                             PP_API_PATH_ALL, params)
+        ret_val, data = self._make_rest_call(action_result, PP_API_PATH_ALL, method='get', params=params)
```

Comment:
The default method for self._make_rest_call() is already `get`

Reply:
Fixed. I think those were in before I rewrote the _make_rest_call function.